maquetado/login.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>login</title>
+  <link rel="stylesheet" type="text/css" href="application.css">
+</head>
+<body>
+  <div class="login-container">
+    <div class="login">
+      <div class="login-logo-container">
+        <img class="login-logo" src="images/LogoWolox.png" alt="logo">
+      </div>
+      <form class="text-roboto">
+        <div class="login-form-text">
+          Email:
+        </div>
+        <input class="login-form-email-input" type="text">
+        <div class="login-form-text">
+          Password:
+        </div>
+        <input class="login-form-password-input" type="text">
+      </form>
+      <div class="login-button-container">
+        <button class="login-button">Login</button>
+      </div>
```

Comment:
Same

Reply:
the idea with this one was that the border radius did not affect the border of the line dividing the login and sign up buttons.